code/client/clrcore/External/WeaponHash.cs

```diff
@@ -104,4 +104,34 @@ public enum VehicleWeaponHash
 		SearchLight = -844344963,
 		Radar = -764006018
 	}
-}
\ No newline at end of file
+
+	public enum AmmoType : ulong
```

Comment:
Relying on sign-extension and ending up with `0xFFFFFFFF...` hashes is a bit weird.
```suggestion
	public enum AmmoType : uint
```